index.html

```diff
@@ -9,8 +9,24 @@
     <title>Ad Data</title>
 </head>
 <body>
-    <canvas id="chart" width="20" height="10"></canvas>
-    <section id="game"></section>
+    <header>
+        <a id="logo" href="index.html">
+            <img src="img/bus.png" alt="home" style="width:200px;height:113px;border:0;">
```

Comment:
You should try and avoid using inline styling to maintain separation of concerns. Just add this to your CSS file.